pydatastructs/miscellaneous_data_structures/priority_queue.py

```diff
@@ -0,0 +1,106 @@
+from heapq import heapify,heappush,heappop
```

Comment:
IMO, we should use `pydatastructs.trees.heaps`

Reply:
Is it necessary to do so? 
Can't we use some external libraries as `heapq` is an inbuilt library in python?